code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -66,1037 +66,1240 @@ Follow by example and make good judgement based on length which list to include
 
 	trimmed
 		name = "Trimmed"
 		icon_state = "hair_trimmed"
 		gender = MALE
-		length = 0
-
-	tightbun
-		name = "Tight Bun"
-		icon_state = "hair_tightbun"
-		gender = FEMALE
-		chatname = "Hair Bun"
+		chatname = "trimmed hair"
 
-	coffeehouse
-		name = "Coffee House Cut"
-		icon_state = "hair_coffeehouse"
+	trimmedflat
+		name = "Trimmed Flat Top"
+		icon_state = "hair_trimmedflat"
 		gender = MALE
-		chatname = "Short Hair"
+		chatname = "trimmed hair"
 
 	undercut
 		name = "Undercut"
 		icon_state = "hair_undercut"
 		gender = MALE
-		chatname = "Short Hair"
+		chatname = "undercut"
 
 	undercut2
 		name = "Undercut 2"
 		icon_state = "hair_undercut2"
 		gender = MALE
+		chatname = "undercut"
 
 	undercut3
 		name = "Undercut 3"
 		icon_state = "hair_undercut3"
 		gender = MALE
+		chatname = "undercut"
 
-	shavedpart
-		name = "Partly Shaved"
-		icon_state = "hair_shavedpart"
-		gender = MALE
-
-	hightight
-		name = "High and Tight"
-		icon_state = "hair_hightight"
-		gender = MALE
-		chatname = "Short Hair"
-
-	topknot
-		name = "Topknot"
-		icon_state = "hair_topknot"
-		gender = MALE
-		length = 3
-		chatname = "Topknot"
-
-	ronin
-		name = "Ronin"
-		icon_state = "hair_ronin"
-		gender = MALE
-		length = 3
-		chatname = "Long Hair"
-
-	bowlcut2
-		name = "Bowl2"
-		icon_state = "hair_bowlcut2"
-		gender = MALE
-		chatname = "Bowl Cut"
-
-	thinning
-		name = "Thinning"
-		icon_state = "hair_thinning"
-		gender = MALE
-		chatname = "Short Hair"
-
-	thinningfront
-		name = "Thinning Front"
-		icon_state = "hair_thinningfront"
-		gender = MALE
-		chatname = "Short Hair"
-
-	thinningback
-		name = "Thinning Back"
-		icon_state = "hair_thinningrear"
+	undercut4
+		name = "Undercut 4"
+		icon_state = "hair_undercut4"
 		gender = MALE
-		chatname = "Short Hair"
+		chatname = "undercut"
 
-	manbun
-		name = "Manbun"
-		icon_state = "hair_manbun"
-		gender = MALE
+	unkept
+		name = "Unkept"
+		icon_state = "hair_unkept"
 		length = 3
-		chatname = "Hair Bun"
-
-	shavedbun
-		name = "Shaved Bun"
-		icon_state = "hair_shavedbun"
-		chatname = "Hair Bun"
-
-
-	halfshaved
-		name = "Half-Shaved"
-		icon_state = "hair_halfshaved"
-		chatname = "Short Hair"
-
-	halfshavedemo
-		name = "Half-Shaved Emo"
-		icon_state = "hair_halfshavedemo"
-		chatname = "Short Hair"
-
-	longsideemo
-		name = "Long Side Emo"
-		icon_state = "hair_longsideemo"
-		chatname = "Side Cut Hair"
-		length = 2
-
-	sideswept
-		name = "Sideswept Hair"
-		icon_state = "hair_sideswept"
-		length = 2
-		chatname = "Sideswept Hair"
-
-	mohawkshaved
-		name = "Shaved Mohawk"
-		icon_state = "hair_mohawkshaved"
-		chatname = "Mohawk"
-
-	mohawkshaved2
-		name = "Tight Shaved Mohawk"
-		icon_state = "hair_mohawkshaved2"
-		chatname = "Mohawk"
-
-	mohawkshavednaomi
-		name = "Naomi Mohawk"
-		icon_state = "hair_mohawkshavednaomi"
-		chatname = "Mohawk"
-
-	leftsidecut
-		name = "Left Sidecut"
-		icon_state = "hair_leftside"
-		chatname = "Side Cut Hair"
+		chatname = "unkept hairdo"

```

Comment:
I really don't know.
The chatname went for "Unkempt", but the name and icon_state go for "Unkept"; the only reason I'm not assuming this is a typo is because the actual icon doesn't look...    'unkempt'. For me, at least.

Reply:
Then rename it.